### ISSUE TITLE
Update S3 object access to maintain connectivity

### ISSUE THREAD

**Issue by ioannakok** [guardian/frontend#26579]
See description in the [email sent by AWS](https://groups.google.com/a/guardian.co.uk/g/dotcom.platform/c/uZIA8iQ3VJQ/m/Ggu2YdKmBAAJ)

Tl;DR; There is a client accessing our `aws-frontend-store` bucket via TLS V1. We need to find and ensure this client is updated to use TLS 1.2 or higher to avoid an availability impact as AWS is sunsetting TLS 1 access.

Here's the data:
Region | Bucket name(s) | APIAction | TLSVersion | NumCalls | UserAgent
-------|----------------|-----------|------------|----------|-----------
eu-west-1 | aws-frontend-store | REST.GET.OBJECT | TLSv1 | 2 | [Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/[102.0.0.0](http://102.0.0.0/) Safari/537.36]

Needs to be done by December 31, 2023

**Comment by rhiannareechaye** [comment]
Can this go on the backlog with a reminder to pick it up in November? maybe we could put it in the diary?

**Comment by mxdvl** [comment]
Not enough traffic to justify fixing, nonce since Oct 5. We suspect this is a crawler.
